src/components/annual-report/section4/SectionFour.js

```diff
@@ -67,46 +68,63 @@ const Section4B = ({ title, subtitle, subsectionData }) => (
-                            <td>{dataRow.divergentSubOrgUnits?.percentage}%</td>
-                        </tr>
-                        <tr>
-                            <td colSpan="2">
-                                {dataRow.divergentSubOrgUnits?.names}
-                            </td>
-                        </tr>
-                    </tbody>
-                </table>
-            ))}
+            .map((dataRow, index) => {
+                return (
+                    <>
+                        <table key={dataRow.name}>
+                            <thead>
+                                <tr>
+                                    <th>{dataRow.name}</th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                <tr>
+                                    <td>Denominator A</td>
+                                    <td>{dataRow.A}</td>
+                                </tr>
+                                <tr>
+                                    <td>Denominator B</td>
+                                    <td>{dataRow.B}</td>
+                                </tr>
+                                <tr>
+                                    <td>Quality threshold</td>
+                                    <td>±{dataRow.qualityThreshold}%</td>
+                                </tr>
+                                <tr>
+                                    <td>Overall score</td>
+                                    <td>{dataRow.overallScore}%</td>
+                                </tr>
+                                <tr>
+                                    <td># Region with poor score</td>
+                                    <td>
+                                        {dataRow.divergentSubOrgUnits?.number}
+                                    </td>
+                                </tr>
+                                <tr>
+                                    <td>% Region with poor score</td>
+                                    <td>
+                                        {
+                                            dataRow.divergentSubOrgUnits
+                                                ?.percentage
+                                        }
+                                        %
+                                    </td>
+                                </tr>
+                                <tr>
+                                    <td colSpan="2">
+                                        {dataRow.divergentSubOrgUnits?.names}
+                                    </td>
+                                </tr>
+                            </tbody>
+                        </table>
+                        <Chart
+                            sectionId={'section4'}
+                            chartId={`chart${index}`}
+                            chartInfo={dataRow.chartInfo}
+                        />
+                    </>
```

Comment:
```suggestion
                    </React.Fragment>
```

Reply:
Fixed in [c1768f9](https://github.com/hisprwanda/who-data-quality-annual-report/pull/37/commits/c1768f9d166b4af39ad9cd71982be8a72b73894e).